tutorials/fluid_modeling.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
<p>If it is important that the user understands the notion of reference components/phases, more information is needed. If not, can it go?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I tried to explain that in the following sentences.</p>